frontend/issues-app/src/app/app.component.ts

```diff
@@ -78,20 +79,20 @@ export class AppComponent {
     })
   }
 
-//   public createNewComment() {
-//     const httpOptions = {
-//       headers: new HttpHeaders({
-//         "Content-Type": "application/json"
-//       })
-//     };
-//     const newComment = {
-//       comment: this.comment
-//     }
+  public createNewComment(issueId) {
+    const httpOptions = {
+      headers: new HttpHeaders({
+        "Content-Type": "application/json"
+      })
+    };
+    const newCommentOnIssue = {
+      comment: this.newComment.text
+    }
 
-//     this.http
-//       .post<string>("http://localhost:9001/issues/comment" + issueId, newComment, httpOptions)
-//       .subscribe(() => {
-//         this.getAllIssues();
-//       });
-// } 
+    this.http
+      .post("http://localhost:9001/issues/" + issueId + "/comment", newCommentOnIssue, httpOptions)
```

Comment:
You should put this url into some kind of config so it's not hard coded. Makes sense?

Reply:
Yes, thanks for advice!